test/e2e/common.go

```diff
@@ -107,6 +107,34 @@ func getEC2Options(t *testing.T) []ec2params.Option {
 	return ec2Options
 }
 
+func (s *linuxInstallerTestSuite) getLatestEmbeddedPythonPath(baseName string) string {
+	vm := s.Env().VM
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
	s.T().Helper()
	vm := s.Env().VM
```